README.md

```diff
@@ -379,18 +423,20 @@ Any additional props are passed to the pressable component.
 |name|type|default|description|
 |:----:|:----:|:----:|:----:|
 |ItemElement|`((props: { name: any; indexDecimal: SharedValue<number>; }) => ReactElement<any, string \| ((props: any) => ReactElement<any, string \| ... \| (new (props: any) => Component<any, any, any>)> \| null) \| (new (props: any) => Component<...>)> \| null) \| (new (props: { ...; }) => Component<...>) \| undefined`|||
+|activeColor|`string \| undefined`|`null`|Color applied to the label when active|
+|inactiveColor|`string \| undefined`|`null`|Color applied to the label when inactive|
 |inactiveOpacity|`number \| undefined`|`0.7`||
 |index|`number`|||
 |indexDecimal|`SharedValue<number>`|||
 |label|`string`|||
-|labelStyle|`StyleProp<AnimateStyle<TextStyle>>`|||
+|labelStyle|`StyleProp<AnimateStyle<TextStyle>>`||Style to apply to the tab item label|
 |name|`any`|||
 |onLayout|`(((event: LayoutChangeEvent) => void) & ((event: LayoutChangeEvent) => void)) \| undefined`||Invoked on mount and layout changes with {nativeEvent: { layout: {x, y, width, height}}}.|
 |onPress|`(name: any) => void`|||
 |pressColor|`string \| undefined`|`#DDDDDD`||
 |pressOpacity|`number \| undefined`|`Platform.OS === 'ios' ? 0.2 : 1`||
 |scrollEnabled|`boolean \| undefined`|||
-|style|`ViewStyle \| (ViewStyle & false) \| (ViewStyle & number & { __registeredStyleBrand: ViewStyle; }) \| (ViewStyle & RecursiveArray<false \| ViewStyle \| RegisteredStyle<...> \| null \| undefined>) \| (ViewStyle & ((state: Readonly<...>) => StyleProp<...>)) \| undefined`||Either view styles or a function that receives a boolean reflecting whether the component is currently pressed and returns view styles.|
+|style|`false \| ViewStyle \| RegisteredStyle<ViewStyle> \| RecursiveArray<false \| ViewStyle \| RegisteredStyle<ViewStyle> \| null \| undefined> \| ... 15 more ... \| undefined`||Either view styles or a function that receives a boolean reflecting whether the component is currently pressed and returns view styles.|
```

Comment:
This looks odd. Might be an issue with doc generation though.

Reply:
The docs script takes the type definition, in the `types.ts` it is `style?: StyleProp<ViewStyle>`, which resolves to this awkward definition. Works in most cases...

Here we don't need it, since the style is already defined in the PressableProps (it is defined twice here). Was my mistake to add it. Also there is an unused prop `ItemElement` that I need to remove from the types (also my mistake). @tcorreiaubi can you please remove both of them from the `types.ts` and then run the docs script?

Reply:
Hey @PedroBern style?: StyleProp<ViewStyle> is required here since the PressableProps omit the style from the viewprops 

```typescript
export interface PressableProps extends AccessibilityProps, Omit<ViewProps, 'style' | 'hitSlop'> {
```

Reply:
You are right! About the weird types in the prop tables, I'm going to solve this later (another day)